packages/sprotty-protocol/src/actions.ts

```diff
@@ -676,16 +676,18 @@ export interface MoveAction extends Action {
     moves: ElementMove[]
     animate: boolean
     finished: boolean
+    stoppable: boolean
```

Comment:
Remark: When you do such protocol changes in futures, keep in mind that there's a Java (Xtend) based implementation, too, see here: https://github.com/eclipse-sprotty/sprotty-server/blob/master/org.eclipse.sprotty/src/main/java/org/eclipse/sprotty/Actions.xtend
I've updated it for now.